src/WatchDogs.FakeSource/DataGenerator.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Bogus;
+
+namespace WatchDogs.FakeSource;
+
+//Deal #1, Balance 10 000, Buy EURUSD 1 lot at 2019-05-12 14:43:12
+//Deal #2, Balance 10 000, Sell GBPUSD 0.2 lots at 2019-05-12 14:43:23
+//Deal #3, Balance 1 000, Sell GBPUSD 1.2 lots at 2019-05-12 14:43:23
+//Deal #4, Balance 10 000, Sell GBPUSD 0.21 lot at 2019-05-12 14:43:24 <- triggered match with deal #2
+//Deal #5, Balance 20 000, Sell GBPUSD 0.4 lot at 2019-05-12 14:43:24 <- triggered match with deal #2 and deal #4
+
+public class DataGenerator : IDataGenerator
```

Comment:
We still need to unit test this class, but I will be forgiving here and I'll create a separate task for that :)

Reply:
no change then